tests/integration/features/integration_tests.feature

```diff
@@ -649,3 +649,26 @@ Feature: Integration tests
         Examples: Local storage
         | storage           | client encryption |
         | local      |  with_client_encryption |
+
+    @17 @skip-cassandra-2
```

Comment:
The reason we skip this test for Cassandra 2.x is that Management API is only supported on Cassandra 3.x and up.